src/packages/engine-core/src/NodeEngine.ts

```diff
@@ -570,6 +570,7 @@ ${chalk.dim("In case we're mistaken, please report this to us 🙏.")}`)
 
         this.port = await this.getFreePort()
         const env = await this.getEngineEnvVars()
+        debug('env', env)
```

Comment:
Do you want to keep this? It's outputting potential secrets to the logs when activated.

Reply:
Good point, will remove it again